tasks/google/add-amazon-linux-2/task.yaml

```diff
@@ -0,0 +1,108 @@
+summary: Add new Amazon linux 2 image to google account
+
+systems: [ubuntu-*]
+
+environment:
+    BUCKET_NAME: spread-images
+    IMAGE_FILE: amazon-linux-2-64
+    IMAGE_URL: https://cdn.amazonlinux.com/os-images/2017.12.0.20180222/kvm/amzn2-kvm-2017.12.0.20180222-x86_64.xfs.gpt.qcow2
+    IMAGE: amazon-linux-2-64-v20180406
+    FAMILY: amazon-linux-2-64
+    DESCRIPTION: Amazon Linux 2 64 bits
+    DISK_NAME: disk.raw     
+
+execute: |
+    . "$TESTSLIB/google.sh"
+
+    # Get the raw image
+    wget -O "${IMAGE_FILE}.qcow2" "$IMAGE_URL"
+    qemu-img convert -O raw "${IMAGE_FILE}.qcow2" "$DISK_NAME"
+
+    # Get /dev/sda1 parition offset:
+    OFFSET=$( sfdisk -J "$DISK_NAME" | python -c 'import sys, json; a=json.load(sys.stdin); print(a["partitiontable"]["partitions"][0]["start"]*512)' )
```

Comment:
Wonder if there should be a check `test -n "$OFFSET"` to rule out cases where the `sfdisk..` incantation failed.